packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionsTableRowAction.tsx

```diff
@@ -33,30 +36,44 @@ export function TransactionsTableRowAction({
 
   const { claim, isClaiming } = useClaimWithdrawal(tx)
   const { claim: claimCctp, isClaiming: isClaimingCctp } = useClaimCctp(tx)
-  const { redeem, isRedeeming } = useRedeemRetryable(tx, address)
+  const { redeem, isRedeeming: isRetryableRedeeming } = useRedeemRetryable(
```

Comment:
We now have 2 redeem flows - one is normal redeem, nothing has been changed there, and if we see the transaction is a teleporter tx, then we have `teleporterRedeem` in the next line. I didn't want to unnecessarily complicate normal redemption flow to handle 2 retryables.